cea/demand/thermal_loads.py

```diff
@@ -1032,7 +1005,14 @@ def calc_prop_rc_model(self, occupancy, architecture, thermal_properties, geomet
         df['GFA_m2'] = df['footprint'] * df['floors']  # gross floor area
         df['Af'] = df['GFA_m2'] * df['Hs']  # conditioned area - areas not heated
         df['Aef'] = df['GFA_m2'] * df['Es']  # conditioned area only those for electricity
-        df['Am'] = df['th_mass'].apply(self.lookup_effective_mass_area_factor) * df['Af']  # Effective mass area in [m2]
+
+
+        if gv.samples:  # if sensitivity analysis is on and there are samples
+            df['Cm'] = df['Cm'] # Internal heat capacity in J/K
```

Comment:
i need help on this one but it will need a better explaination, i can do it when we meet